decidim-ai/spec/shared/events_examples.rb

```diff
@@ -39,6 +39,17 @@
       expect(Decidim::Report.count).to eq(spam_count)
     end
   end
+
+  it "hides the resource" do
```

Comment:
```suggestion
  it "hides automatically the resource" do
```

Reply:
Can you also add the counter-example? (When the setting is false, then it doesn't hides the resource)